### PR TITLE
Set up Email 1 w/ fixes across all emails

### PR DESCRIPTION
**Pull request by rememberlenny**
https://trello.com/c/6LxCjA2T/210-1-round-3

# Note

Fix a centering issue across templates on mobile

screen shot 2016-03-11 at 2 39 25 pm
screen shot 2016-03-11 at 2 39 18 pm

### CODE REVIEW

**Comment by jessieay** [comment]
@rememberlenny I don't see the approval chain stuff in this email, but it is in the screenshot in the trello card..

File: app/models/ncr/work_order.rb [hunk 1]
```diff
@@ -134,7 +134,7 @@ def not_ba60?
     def total_price
       amount || 0.0
     end
-
+
```
Comment (jessieay) [inline app/models/ncr/work_order.rb]: :heart_eyes:

Comment (pkarman) [inline app/models/ncr/work_order.rb]: heh I didn't realize we were already normalizing on `total_price`

let's pick one or the other. creating new clients will make this a PITA to keep sync'd.

File: app/models/gsa18f/procurement.rb [hunk 2]
```diff
@@ -64,6 +64,10 @@ def total_price
       (cost_per_unit * quantity) || 0.0
     end
 
+    def amount
```
Comment (jessieay) [inline app/models/gsa18f/procurement.rb]: I might suggest also adding the `amount` method to the `ProposalDecorator` and then calling `@proposal.amount` in the view. you can define `amount` in the decorator like this:

```ruby
def amount
  client_data.try(:amount) || ""
end
```

then you will also need to define `amount` on `WorkOrder` -- the benefit of this is that when a proposal doesn't have any client data (which won't happen in real life but does happen in tests) you won't deal with a `nil` error

Comment (jessieay) [inline app/models/gsa18f/procurement.rb]: also see https://github.com/18F/C2/pull/1052/files#r55882651 -- maybe use `total_price` instead?

**Comment by rememberlenny** [comment]
@jessieay Re: approval chain, it was a local issue. Things are alright!

**Comment by jessieay** [comment]
@rememberlenny made some changes here d82d4ec and committed to your branch -- spec problems should be fixed now (issue was in tests that created a proposal without client data...always nice to make things less dependent on each other so created a method on the proposal decorator that returns an empty string when there is no client data)

**Comment by rememberlenny** [comment]
@jessieay Thanks!

**Pull request merged**